### PR TITLE
chore(docs): remove typo in store README

### PR DESCRIPTION
**Pull request by peterbsmith2**

### CODE REVIEW

File: docs/store/README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ export class MyAppComponent {
   count$: Observable<number>;
 
   constructor(private store: Store<AppState>) {
-    this.count$ = store.pipe(select('count'));
+    this.count$ = this.store.pipe(select('count'));
```
Comment (brandonroberts) [inline docs/store/README.md]: Why the change? Either way is valid since its in the constructor.

Comment (peterbsmith2) [reply docs/store/README.md]: @brandonroberts I was running examples last night and encountered an error. I may have been at less than my best. So you're right and I think this is unnecessary now.
